admin/commands/storage/read_transactions.go

```diff
@@ -60,23 +60,26 @@ func usageErr(msg string) error {
 	return fmt.Errorf("required flags \"start-height\", \"end-height\", %s", msg)
 }
 
+// Returns admin.InvalidAdminReqError for invalid inputs
 func findUint64(input map[string]interface{}, field string) (uint64, error) {
 	data, ok := input[field]
 	if !ok {
-		return 0, usageErr(fmt.Sprintf("%s not set", field))
+		return 0, admin.NewInvalidAdminReqErrorf("missing required field '%s'", field)
 	}
 	val, err := parseN(data)
 	if err != nil {
-		return 0, usageErr(fmt.Sprintf("%s must be a uint64 value, but got %v: %v", field, data, err))
+		return 0, admin.NewInvalidAdminReqErrorf("invalid 'n' field: %w", err)
```

Comment:
better to print the field as well

Reply:
Done in https://github.com/onflow/flow-go/pull/3391/commits/10bc2b1e60723fdbf79d926c382d9d205f4b032f